modules/articles.py

```diff
@@ -0,0 +1,59 @@
+import newspaper
+import re
+
+class article_content():
+
+    def __init__(self, article_object):
+        #Whenever a class object is made, the article is tokenized and the dictionary file is opened as a set.
+        self.tokenized_words = nltk.word_tokenize(article_object.text)
+        self.article_object = article_object
+        with open('words_alpha.txt') as word_file:
```

Comment:
Why is your dictionary not a module input? That would be significantly faster and easier.

Reply:
Can you elaborate?

Reply:
For example; https://pypi.org/project/PyDictionary/
Using mapping instead of iterating is significantly faster.

Reply:
Well, we are only going to check whether the word exists in the file or not...don't really want to extract any other definition.

Sets are quite fast at determining whether an element in present in them or not because sets can only have unique values (it was the fastest method that I tried by quite a long margin). I'll have to compare sets vs mapping after I read up a little bit more on mapping. Still quite unsure about what it perfectly means. 

I do believe just making the set once is enough though, so you like suggest, maybe it can be done in a different file and then imported when necessary.

Reply:
Why would you want it to exist in a file? It needs to be in the dictionary since it needs to be an actual word. The fastest way to check if a word is in the dictionary is using a dictionary module.